query/action.yml

```diff
@@ -1,22 +1,26 @@
-name: 'CodeQL: Query'
-description: 'Run a CodeQL query against a repository'
-author: 'GitHub'
+name: "CodeQL: Query"
+description: "Run a CodeQL query against a repository"
+author: "GitHub"
 inputs:
   query:
-    description: 'Text of CodeQL query to run'
+    description: "Text of CodeQL query to run. Exactly one of 'query' and 'query_pack_url' are required."
+    required: true
+
+  query_pack_url:
+    description: "URL for a base64 encoded tarball containing CodeQL query plus dependencies. Exactly one of 'query' and 'query_pack_url' are required."
```

Comment:
Just checking what exactly "URL for a base64 encoded tarball" means and is it correct? It seems the base64 encoding is unnecessary if it's going to be downloaded via a URL.

If this is just the way it is and changing it would be effort then that's fine. Just wanted to check as it seemed odd to me.

Reply:
It's because the API call passes it in the JSON body as a base64 string. We could decode it in the API handler before inserting it into S3, but that doesn't happen currently. Do you think that we should make that change?

If not, you've reminded me that I need to do the base64 decoding here!

Reply:
I agree it makes sense to do the decoding here rather than in the API handler. So leaving it as it is (i.e. base64 encoded) is fine 👍